src/components/Dialogs/MoreAbout.jsx

```diff
@@ -2,46 +2,58 @@ import React from 'react'
-        <div className="header layout-row">
-          <h3 className="flex-all">More about the complaint database</h3>
+        <div className="header">
+          <h3>
+            Things you should know before you use this database
+          </h3>
         </div>
         <div className="body">
           <p className="body-copy">
-            We don’t verify all the facts alleged in these complaints, but we
-            do give companies the opportunity to confirm they have a commercial
-            relationship with the consumer before we publish the complaint in
-            this database.
+            This database is not a statistical sample of consumers’ experiences
+            in the marketplace.
+            Complaints are not necessarily representative of all consumers’
+            experiences with a financial product or company.
           </p>
           <p className="body-copy">
-            We only publish the description of what happened if the consumer
-            agrees to share it and after we take steps to&nbsp;
+            The database only includes complaints the CFPB sent to companies and
+            complaints are only published after the company responds, confirming
+            a commercial relationship or after 15 days, whichever comes first.
+            The database does not include complaints referred to other
+            regulators, such as complaints about depository institutions with
+            less than $10 billion in assets. Company-specific information should
+            be considered in the context of that company’s size and/or market
+            share.
+          </p>
+          <p className="body-copy">
+            We only publish a consumer’s description of what happened in their
+            own words if the consumer agrees to share it and after we take steps
+            to&nbsp;
             <a href={urlScrubbing}
                target="_blank"
                rel="noopener noreferrer">
-              remove personal information.
-            </a>
+              remove personal information
+            </a>.&nbsp;
+            One consumer’s experience is not necessarily representative of all
+            consumers’ experiences and narratives are not verified before
+            publication.
           </p>
           <p className="body-copy">
-            We don’t publish complaints if they don't meet our&nbsp;
+            The database generally updates daily. We don’t publish complaints if
+            they don’t meet our&nbsp;
             <a href={urlCriteria}
                target="_blank"
                rel="noopener noreferrer">
-              publication criteria.
-            </a>
-            &nbsp;Data is generally updated daily.
-          </p>
-          <p className="body-copy">
-            Company specific information should be considered in the context of
-            that company's size and/or market share.
+              publication criteria
+            </a>.
```

Comment:
@anselmbradford, sorry for the churn, but this content has been rearranged slightly and some of the links have been updated. Here's the new content:

> ## Things you should know before you use this database
> 
> This database only includes complaints the CFPB sent to companies and complaints are only published after the company responds, confirming a commercial relationship or after 15 days, whichever comes first. This database does not include complaints referred to other regulators, such as complaints about depository institutions with less than $10 billion in assets. 
> 
> This database is not a statistical sample of consumers’ experiences in the marketplace. Complaints are not necessarily representative of all consumers’ experiences with a financial product or company. Company-specific information should be considered in the context of that company’s size and/or market share.
> 
> This database only includes a consumer’s description of what happened in their own words if the consumer agrees to share it and after we take steps to [remove personal information](https://files.consumerfinance.gov/f/documents/201503_cfpb_Narrative-Scrubbing-Standard.pdf). One consumer’s experience is not necessarily representative of all consumers’ experiences and narratives are not verified before publication.
> 
> This database generally updates daily and doesn’t include complaints that don’t meet our [publication criteria](https://files.consumerfinance.gov/f/documents/201503_cfpb_disclosure-of-consumer-complaint-narrative-data.pdf).

Reply:
Updated in https://github.com/cfpb/ccdb5-ui/pull/387/commits/d891910a16d6dd21dfa5b013adde7a1bc4dd820e

<img width="518" alt="Screen Shot 2022-02-15 at 4 07 38 PM" src="https://user-images.githubusercontent.com/704760/154149417-46e5da87-00d5-494a-9d1d-be7019db73fc.png">